### PR TITLE
Rework Downloads to Allow Exports of Large Regions

### PR DESCRIPTION
**Pull request by rush42**
TODO:

- [ ] find a way to inlcude the attribution and license

### CODE REVIEW

**Comment by rush42** [comment]
[FlatGeoBuf](http://flatgeobuf.org/) has no top-level attributes. Because we currently have the attribution and license in the top-level attributes (in the FeatureCollection) we need an alternative solution to inform about attribution and license.
I currently have two solutions in mind:

1. export a zip archive which contains the data as `.fbg` file and a `license.md`
2. add the license to each objects attributes

**Comment by rush42** [comment]
I'll merge this anyway to test the performance on larger regions.

File: app/src/instrumentation/initExportFunctions.ts [hunk 1]
```diff
@@ -5,46 +5,50 @@ import {
...
+      const metaKeyQuery: Array<{ key: string }> = await prismaClientForRawQueries.$queryRawUnsafe(`
+        SELECT DISTINCT jsonb_object_keys(meta) AS key
+        FROM public."${tableName}"
+      `)
+      const tagKeys = tagKeyQuery.map(({ key }) => `tags->>'${key}' as "${key}"`).join(',')
+      const metaKeys = metaKeyQuery.map(({ key }) => `meta->>'${key}' as "${key}"`).join(',')
 
       return prismaClientForRawQueries.$transaction([
-        prismaClientForRawQueries.$executeRaw`SET search_path TO public;`,
+        prismaClientForRawQueries.$executeRaw`SET search_path public;`,
```
Comment (tordans) [inline app/src/instrumentation/initExportFunctions.ts:27]: Vielleicht obsolet, da wir das "public" durch den anderen PR gelöscht haben; aber dass hier kein "TO" steht, kommt mir komisch vor; unten steht es noch.

File: psql_shell.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+source ./.env
+docker exec -ti db psql -d $DATABASE_NAME -U $DATABASE_USER
```
Comment (tordans) [inline psql_shell.sh:3]: Bitte File kommentieren. Was macht das? Was ist der Kontext?

**Review by tordans — COMMENTED** [review]
@rush42 bitte schau nochmal in die Comments. Merci.

**Pull request merged**